.scripts/list-of-samples.json

```diff
@@ -9,12 +9,14 @@
     "custom-logger",
     "dsl-interpreter",
     "ejson",
+    "empty",
     "encryption",
     "expense",
     "fetch-esm",
     "food-delivery",
     "grpc-calls",
     "hello-world",
+    "hello-world-esm",
```

Comment:
remove this and the dir.

also, you also needed this change:

https://github.com/temporalio/samples-typescript/pull/327/files#diff-88397926b71aad9ca766a35d4c240f95914117d55d5521d57ebe2ad7b0a390c2

before committing (the commit hook wrote/overwrote some files it shouldn't have in this PR).